views/layout.erb

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Checkpoint 05</title>
+    <link rel="stylesheet" href="styles.css">
```

Comment:
I believe you need the `/` at the front to make this a relative path otherwise it wont work on every view.
i.e. `/styles.css`